src/pages/Overview/FiltersAndSorts.ts

```diff
@@ -0,0 +1,76 @@
+import { FILTER_ACTION_APPEND, FilterType, FilterValue } from '../../types/Filters';
+import { FAILURE, DEGRADED, HEALTHY } from '../../types/Health';
+import { NamespaceInfo } from './NamespaceInfo';
+
+export namespace FiltersAndSorts {
+  export const nameFilter: FilterType = {
+    id: 'namespace_search',
+    title: 'Name',
+    placeholder: 'Filter by Name',
+    filterType: 'text',
+    action: FILTER_ACTION_APPEND,
+    filterValues: []
+  };
+
+  const healthValues: FilterValue[] = [
+    {
+      id: FAILURE.name,
+      title: FAILURE.name
+    },
+    {
+      id: DEGRADED.name,
+      title: DEGRADED.name
+    },
+    {
+      id: HEALTHY.name,
+      title: HEALTHY.name
+    }
+  ];
+
+  export const healthFilter: FilterType = {
+    id: 'health',
+    title: 'Health',
+    placeholder: 'Filter by Application Health',
+    filterType: 'select',
+    action: FILTER_ACTION_APPEND,
+    filterValues: healthValues
+  };
+
+  export const availableFilters: FilterType[] = [nameFilter, healthFilter];
+
+  export interface SortField {
+    id: string;
+    title: string;
+    isNumeric: boolean;
+    param: string;
+    compare: (a: NamespaceInfo, b: NamespaceInfo) => number;
+  }
+
+  export const sortFields: SortField[] = [
+    {
+      id: 'namespace',
```

Comment:
id is 'namespace' but title is 'Name' is this on purpose ?
I guess this is to sort by name, isn't it ?

Reply:
Yes by name, but it's the namespace name. I find "namespace" more accurate for an id. But it's displaying "Name" as per UXD mockup.